### PR TITLE
Immutable diamond

### PR DESCRIPTION
**Pull request by 0xDEnYO**

### CODE REVIEW

File: script/UpdateAmarokFacet.s.sol [hunk 1]
```diff
@@ -5,7 +5,9 @@ import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
 import { stdJson } from "forge-std/StdJson.sol";
 import "forge-std/console.sol";
 import { DiamondCutFacet, IDiamondCut } from "lifi/Facets/DiamondCutFacet.sol";
+import { OwnershipFacet } from "lifi/Facets/OwnershipFacet.sol";
```
Comment (ezynda3) [inline script/UpdateAmarokFacet.s.sol:8]: What's this imported for?

File: script/UpdateAmarokFacet.s.sol [hunk 2]
```diff
@@ -5,7 +5,9 @@ import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
...
 import { AmarokFacet } from "lifi/Facets/AmarokFacet.sol";
+import {DSTest} from "ds-test/test.sol";
```
Comment (ezynda3) [inline script/UpdateAmarokFacet.s.sol:10]: And this?

File: script/UpdateAmarokFacet.s.sol (hunk 1 above)
Comment (0xDEnYO) [reply script/UpdateAmarokFacet.s.sol:8]: removed

File: script/UpdateAmarokFacet.s.sol (hunk 2 above)
Comment (0xDEnYO) [reply script/UpdateAmarokFacet.s.sol:10]: removed

**Pull request merged**